build.sbt

```diff
@@ -228,7 +229,7 @@ lazy val cassandra = project
   .settings(commonSettings)
   .settings(Settings.cassandra)
 
-lazy val hbase = project
+lazy val `hbase-store` = project
```

Comment:
Since the package path is `geotrellis.store.hbase` wouldn't it make more sense to call the project `store-hbase`?

Reply:
I'm not sure I find that reason compelling, but I think `store-hbase` is likely to lend itself to better ordering of modules by default

Reply:
That's true. I'm fine with leaving it `store-xxx`. Thought I'd just mention it.